### ISSUE TITLE
Providing aws_access_key , aws_secret_key does not work

### ISSUE THREAD

**Issue by fvant** [hsingh/ansible-elastic-beanstalk#18]
All AWS modules support the use of these parameters but this module does not.
They are "parsed" at the module init but not actually used to connect.  
See this snippet from the ELB module that demonstrates the right way:

```
    def _get_elb_connection(self):
        try:
            return connect_to_aws(boto.ec2.elb, self.region, **self.aws_connect_params)
```

or the vpc_nacl module using boto3:

```
    try:
        region, ec2_url, aws_connect_kwargs = get_aws_connection_info(module, boto3=True)
        client = boto3_conn(module, conn_type='client', resource='ec2', region=region, endpoint=ec2_url, **aws_connect_kwargs)
    except botocore.exceptions.NoCredentialsError, e:
        module.fail_json(msg="Can't authorize connection - "+str(e))
```

**Comment by sidewinder12s** [comment]
@fvant This might not be an issue with the boto3 re-write. I don't use static keys so I don't have anything readily available to test.

**Comment by hsingh** [comment]
As mentioned by @sidewinder12s this is fixed in version 2.0 and higher. Please verify.
